### DISCUSSION TITLE
How to get login and password to connect to the table?

### DISCUSSION THREAD

**Discussion by artemdzyuba** [supabase/supabase#2841] [Questions] [answered]
I try to connect to the base from outside, using n8n and appsmith. I'm not familiar with PostgreSQL auth issues.

So they demand User and Password to the database. I hasn't found anything about usernames in Supabase docs. I see that the use username "postgres" (see [here](https://youtu.be/9va1dofeiFA?t=1231) for example).  Where should I get this credentials for my Supabase database?

Thank you in advance.

**Comment by soedirgo** [comment, marked as answer]
Hey @artemdzyuba, the user is `postgres` the password is the one you provided when you created your project.

**Reply by artemdzyuba** [reply]
Thank you!

**Reply by Dal-Tron** [reply]
Do you know how I can reset this password if I lost it?

**Reply by Dal-Tron** [reply]
Turns out you can set it from the SQL editor:
```
alter user postgres 
with password 'new_password';
```
